### PR TITLE
Updating TOC to make content more findable.

### PR DESCRIPTION
**Pull request by jonquilwilliams**

### CODE REVIEW

**Comment by jonquilwilliams** [comment]
Peer review needed. This is just a splitting out of the Getting Started/Creating Your First Cluster assembly into smaller assemblies that make the TOC navigation easier.

JIRA card: https://issues.redhat.com/browse/OSDOCS-1655

Manual preview here: http://file.rdu.redhat.com/jonwilli/moa/JIRA1655/getting_started_rosa/quickstart-rosa.html

**Comment by jonquilwilliams** [comment]
Also, this is should be merged against the 4.5 branch since it's for ROSA. :)

File: getting_started_rosa/quickstart-rosa.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[id="quickstart-rose"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+:context: quickstart-rosa
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `rosa`, the {product-title} command line utility.
```
Comment (bobfuru) [inline getting_started_rosa/quickstart-rosa.adoc:8]: s/This guide walks through/This guide walks you through/
s/command line utility/command-line utility/

File: getting_started_rosa/quickstart-rosa.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+include::modules/rosa-quickstart-instructions.adoc[leveloffset=+1]
+
+For additional help, consult the rest of this guide. By the end of this guide you will have an {product-title} cluster running in your AWS account.
```
Comment (bobfuru) [inline getting_started_rosa/quickstart-rosa.adoc:12]: s/guide you will/guide, you will/

File: modules/rosa-create-cluster-admins.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-create-dedicated-cluster-admins.adoc"]
+= Granting cluster-admin access
```
Comment (bobfuru) [inline modules/rosa-create-cluster-admins.adoc:7]: s/cluster-admin/`cluster-admin`/

File: modules/rosa-create-cluster-admins.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. These privileges are not automatically assigned to your user account when you create the cluster.
```
Comment (bobfuru) [inline modules/rosa-create-cluster-admins.adoc:9]: s/user that/user who/

File: modules/rosa-create-cluster-admins.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+Additionally, only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges. As a best practice, limit the number of `cluster-admin` users.
```
Comment (bobfuru) [inline modules/rosa-create-cluster-admins.adoc:11]: Per ISG, I would reword the second sentence so that it does not start lowercase. Perhaps:
> Users with `dedicated-admin` access have fewer privileges.

Comment (bobfuru) [reply modules/rosa-create-cluster-admins.adoc:11]: Limit the number to what?

File: modules/rosa-create-cluster-admins.adoc [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+.Prerequisites
+
+* You have added an Identity Provider (IDP) to your cluster.
```
Comment (bobfuru) [inline modules/rosa-create-cluster-admins.adoc:15]: I don't think "Identity Provider" needs to be capitalized here as a generic term.

File: modules/rosa-create-cluster-admins.adoc [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+----
++
+. Give your user `cluster-admin` privileges:
++
+[source,terminal]
+----
+$ rosa grant user cluster-admin --user <idp_user_name> --cluster <cluster_name>
+----
++
+. Verify your user is listed as a `cluster-admin`:
```
Comment (bobfuru) [inline modules/rosa-create-cluster-admins.adoc:35]: I would spell out "cluster administrator" here where it is in generic noun form (as opposed to where you use it as an adjective elsewhere, such as _`cluster-admin` privileges_).

File: modules/rosa-create-cluster-admins.adoc [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
++
+.Example output
+[source,terminal]
+----
+GROUP             NAME
+cluster-admins    rh-rosa-test-user
+dedicated-admins  rh-rosa-test-user
+----
++
+. Enter the following command to verify that your user now has `cluster-admin` access. As a `cluster-admin` you should be able to enter the following command without errors.
```
Comment (bobfuru) [inline modules/rosa-create-cluster-admins.adoc:50]: Is there a way to combine these sentences so you are not saying "enter the following command" twice?
> As a cluster administrator, you can verify that your user now has `cluster-admin` access by entering the following command:

(I'm not sure that you need the part about not getting errors.)

File: modules/rosa-create-dedicated-cluster-admins.adoc [hunk 9]
```diff
@@ -4,8 +4,8 @@
 
 
 [id="rosa-create-dedicated-cluster-admins.adoc"]
-= Granting dedicated-admin and cluster-admin access
-As the user that created the cluster, add the `cluster-admin` user role to your account to have the maximum administrator privileges. Additionally, only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges.
+= Granting dedicated-admin access
+Only the user who created the cluster can grant cluster access to other `cluster-admin` or `dedicated-admin` users. `dedicated-admin` users have fewer privileges. As a best practice, grant `dedicated-admin` access to most of your administrators.
```
Comment (bobfuru) [inline modules/rosa-create-dedicated-cluster-admins.adoc:8]: Same comment as above about not starting sentence with lowercase.

File: modules/rosa-create-dedicated-cluster-admins.adoc [hunk 10]
```diff
@@ -15,15 +15,14 @@ As the user that created the cluster, add the `cluster-admin` user role to your
 
 .Procedure
 
-. To grant cluster access to a `dedicated-admin` user:
-.. Enter the following command to promote your user to a `dedicated-admin`:
+. Enter the following command to promote your user to a `dedicated-admin`:
```
Comment (bobfuru) [inline modules/rosa-create-dedicated-cluster-admins.adoc:18]: `dedicated admin` -> dedicated administrator

File: modules/rosa-create-dedicated-cluster-admins.adoc [hunk 11]
```diff
@@ -41,53 +40,3 @@ dedicated-admins   rh-rosa-test-user
 ====
 A Forbidden error displays if user without `dedicated-admin` privileges runs this command.
```
Comment (bobfuru) [inline modules/rosa-create-dedicated-cluster-admins.adoc:41]: While you're in here:
s/A Forbidden error displays if user/A `Forbidden` error displays if a user/

File: modules/rosa-create-dedicated-cluster-admins.adoc [hunk 12]
```diff
@@ -15,15 +15,14 @@ As the user that created the cluster, add the `cluster-admin` user role to your
...
-.. Enter the following command to promote your user to a `dedicated-admin`:
+. Enter the following command to promote your user to a `dedicated-admin`:
 +
 [source,terminal]
 ----
 $ rosa grant user dedicated-admin --user <idp_user_name> --cluster <cluster_name>
 ----
 +
-.. Enter the following command to verify that your user now has `dedicated-admin` access. As a `dedicated-admin` you should see output similar to the example output when entering the following command:
+. Enter the following command to verify that your user now has `dedicated-admin` access. As a `dedicated-admin` you should see output similar to the example output when entering the following command:
```
Comment (bobfuru) [inline modules/rosa-create-dedicated-cluster-admins.adoc:25]: `dedicated admin` -> dedicated administrator
Per ISG, avoid "should"...and I don't think you need the second sentence at all.

File: _topic_map.yml [hunk 13]
```diff
@@ -60,8 +60,18 @@ Name: Getting started
 Dir: getting_started_rosa
 Distros: openshift-rosa
 Topics:
+- Name: Quickstart
```
Comment (bobfuru) [inline _topic_map.yml:63]: Is there a reason we are using "Quickstart" as one word and not two here and in `modules/rosa-quickstart-instructions.adoc`? Looks odd to me, but I find that we also have something in OCP called a "Quickstart template" (https://docs.openshift.com/container-platform/4.6/openshift_images/using-templates.html#templates-using-instant-app-quickstart_using-templates)

**Review by bobfuru — CHANGES_REQUESTED** [review]
Some comments and questions, mostly about style and formatting. Let me know when you're ready for another look.

File: getting_started_rosa/quickstart-rosa.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply getting_started_rosa/quickstart-rosa.adoc:8]: Done.

File: modules/rosa-create-cluster-admins.adoc (hunk 3 above)
Comment (jonquilwilliams) [reply modules/rosa-create-cluster-admins.adoc:7]: Done.

File: modules/rosa-create-cluster-admins.adoc (hunk 4 above)
Comment (jonquilwilliams) [reply modules/rosa-create-cluster-admins.adoc:9]: Done.

File: modules/rosa-create-cluster-admins.adoc (hunk 5 above)
Comment (jonquilwilliams) [reply modules/rosa-create-cluster-admins.adoc:11]: Done. Limit to as few as possible. One of the things we're trying to do is get better guidance around cluster-admin versus dedicated admins on OSD, and the specific differences between the privileges. Future enhance,ent. This is the best I can do for now.

File: getting_started_rosa/quickstart-rosa.adoc (hunk 2 above)
Comment (jonquilwilliams) [reply getting_started_rosa/quickstart-rosa.adoc:12]: Done.

File: modules/rosa-create-dedicated-cluster-admins.adoc (hunk 9 above)
Comment (jonquilwilliams) [reply modules/rosa-create-dedicated-cluster-admins.adoc:8]: Done.

File: modules/rosa-create-dedicated-cluster-admins.adoc (hunk 11 above)
Comment (jonquilwilliams) [reply modules/rosa-create-dedicated-cluster-admins.adoc:41]: Done.

File: modules/rosa-create-dedicated-cluster-admins.adoc (hunk 12 above)
Comment (jonquilwilliams) [reply modules/rosa-create-dedicated-cluster-admins.adoc:25]: Done.

File: _topic_map.yml (hunk 13 above)
Comment (jonquilwilliams) [reply _topic_map.yml:63]: No reason, I inherited that and thought it was our standard. And it was probably named that because of the template. Anyway, I have changed it to two words. Thanks!

**Comment by jonquilwilliams** [comment]
@bobfuru Updated manual preview here: http://file.rdu.redhat.com/jonwilli/moa/JIRA1655/getting_started_rosa/quickstart-rosa.html

Thank you for your helpful feedback.

File: modules/rosa-create-cluster-admins.adoc (hunk 6 above)
Comment (jonquilwilliams) [reply modules/rosa-create-cluster-admins.adoc:15]: Done.

File: modules/rosa-create-cluster-admins.adoc (hunk 8 above)
Comment (jonquilwilliams) [reply modules/rosa-create-cluster-admins.adoc:50]: Done. I removed the redundancy and clarified why that dedicated administrators will get an error but cluster administrators will not. Our users will need to know that difference in privileges.

**Review by adellape — APPROVED** [review]
Latest changes LGTM. :+1:
